hazelcast/generated-sources/src/hazelcast/client/protocol/codec/codecs.cpp

```diff
@@ -132,7 +117,7 @@ namespace hazelcast {
                 }
 
                 ClientMessage client_createproxy_encode(const std::string &name, const std::string &service_name) {
-                    size_t initial_frame_size = ClientMessage::REQUEST_HEADER_LEN;
+                    size_t initial_frame_size = ClientMessage::REQUEST_HEADER_LEN ;
```

Comment:
There is an extra space here

Reply:
These existed in the protocol repo and was deleted manually here. They came back since I regenerated. I fixed this in the protocol repo (see https://github.com/hazelcast/hazelcast-client-protocol/pull/410/commits/3a0bf963ffc07c5065cb94a2f3e35b6ce697bdd2) and now it's fine.